src/mgr/PyModuleRegistry.h

```diff
@@ -122,11 +122,17 @@ class PyModuleRegistry
     return modules.at(module_name);
   }
 
-  bool module_exists(const std::string &module_name) const
+  /**
+   * Find the specified module.
+   * Returns a null reference if it does not exist.
+   */
+  PyModuleRef find_module(const std::string &module_name)
   {
     std::lock_guard l(lock);
-    auto mod_iter = modules.find(module_name);
-    return mod_iter != modules.end();
+    if (!modules.count(module_name)) {
+      return nullptr;
```

Comment:
nit: I would prefer return `PyModuleRef()` or `{}`, instead of `nullptr` being implicitly converted to `PyModuleRef`.